### ISSUE TITLE
Latest version not installing monotonic

### ISSUE THREAD

**Issue by camtr0n** [spectacles-ci/spectacles#760]
Our CI runs are failing due to missing `monotonic` module after using `pip install spectacles` to set up our environment. Seeing this locally as well.

Failure:
```
Run bash ../etc/ci.run-spectacles-lookml.sh prod
Running Spectacles: LookML validation
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.18/x64/bin/spectacles", line 5, in <module>
    from spectacles.cli import main
  File "/opt/hostedtoolcache/Python/3.9.[18](https://github.com/replicatedhq/data/actions/runs/7738160085/job/21098442542#step:5:19)/x64/lib/python3.9/site-packages/spectacles/cli.py", line [19](https://github.com/replicatedhq/data/actions/runs/7738160085/job/21098442542#step:5:20), in <module>
    import spectacles.tracking as tracking
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/spectacles/tracking.py", line 5, in <module>
    import analytics
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/analytics/__init__.py", line 3, in <module>
    from analytics.client import Client
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/analytics/client.py", line 11, in <module>
    from analytics.consumer import Consumer
  File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/analytics/consumer.py", line 3, in <module>
    import monotonic
ModuleNotFoundError: No module named 'monotonic'
Error: Process completed with exit code 1.
```

Will paste `pip install spectacles` output into a comment

**Comment by camtr0n** [comment]
current workaround is to update all our workflows to use `pip install spectacles monotonic==1.6`

**Comment by joshtemple** [comment]
Thanks @camtr0n for the report. This is resolved in `v2.4.1`.

**Comment by joshtemple** [comment]
This actually seems to have created a different issue, also with the `segment-analytics-python` package. Pinning to a previous version should fix it, but I'm going to yank `2.4.1` from PyPI while I make this next fix.

**Comment by joshtemple** [comment]
Okay, `v2.4.2` should do it. Let me know if you have any issues installing.
